mirroroperator/registrymirror.py

```diff
@@ -255,7 +255,7 @@ def generate_daemon_set(self, daemon_set):
         return daemon_set
 
     def generate_service(self, service):
-        service.spec.type = "NodePort"
+        service.spec.type = "ClusterIP"
```

Comment:
is this entire method strictly necessary ? Isn't it a ClusterIP by default ?

Reply:
(I have deleted this entire method on my branch, which I have not pushed to github yet)